tools/remote_build/rbe_common.bazelrc

```diff
@@ -87,8 +87,3 @@ build:ubsan --copt=-gmlt
 # use double the default value for "moderate" and "long" timeout as sanitizer
 # tests tend to be slower
 build:ubsan --test_timeout=60,600,1800,3600
-# override the config-agnostic crosstool_top
```

Comment:
not that ubsan RBE tests are now basically disabled (see b/200667821), but based on my understanding, making this change doesn't aggravate the current situation.  Re-enabling ubsan can be handled separately, and this change making here won't make that harder to fix.